app/src/main/java/com/example/mycalculator/symbols/OperatorsSymbols.java

```diff
@@ -0,0 +1,8 @@
+package com.example.mycalculator.symbols;
+
+public class OperatorsSymbols {
+    public static final String operatorSum = "+";
```

Comment:
The constant names should be in upper case and the words separated by an underscore
OPERATOR_SUM

Reply:
Done